i3lock.1

```diff
@@ -110,6 +112,19 @@ Show the number of failed attempts, if any.
 Enables debug logging.
 Note, that this will log the password used for authentication to stdout.
 
+.TP
+.BI \-C\  path \fR,\ \fB\-\-crash-handler= path
+Runs the specified command on unexpected process termination (e. g. crash).
+
+For example to log out cleanly:
```

Comment:
Examples are nice, but this one is extremely specific to the system you're using and will not work for many people. I fear it might cause more questions than it does good. Using a systemd command for a forced shutdown might be more widely applicable and also more concise.